test/jdk/jdk/incubator/vector/DoubleMaxVectorTests.java

```diff
@@ -609,7 +625,6 @@ static void assertDoubleBroadcastArraysEquals(double[] r, double[] a, double[] b
         }
     }
 
-
```

Comment:
This empty line should not get removed.

Reply:
Fixed.
It is indeed there are 2 empty lines at https://github.com/openjdk/jdk/blob/e6f23a90d4a53339a3c9c2b76fc5d317940e4472/test/jdk/jdk/incubator/vector/DoubleMaxVectorTests.java#L720-L721